src/ai/backend/common/docker.py

```diff
@@ -360,17 +360,17 @@ class PlatformTagSet(Mapping):
     _data: dict[str, str]
     _rx_ver = re.compile(r"^(?P<tag>[a-zA-Z]+)(?P<version>\d+(?:\.\d+)*[a-z0-9]*)?$")
 
-    def __init__(self, tags: Iterable[str]):
+    def __init__(self, tags: Iterable[str], value: str = None):
```

Comment:
As we update this code, let's just add a missing type annotation as a part of small refactoring :wink::
```suggestion
    def __init__(self, tags: Iterable[str], value: str = None) -> None:
```